cpp/modmesh/view/wrap_view.cpp

```diff
@@ -35,6 +35,103 @@
 #include <QPointer>
 #include <QClipboard>
 
+#ifdef MODMESH_USE_PYSIDE
+
+// Usually MODMESH_PYSIDE6_FULL is not defined unless for debugging.
+#ifdef MODMESH_PYSIDE6_FULL
+#include <pyside.h>
+#else // MODMESH_PYSIDE6_FULL
+#define PYSIDE_API MODMESH_PYTHON_WRAPPER_VISIBILITY
+namespace PySide
+{
+// The prototypes are taken from pyside.h
```

Comment:
Instead of directly include PySide/Shiboken headers, I transcribe the prototype here.  They should be pretty stable already.